drivers/nrf24l01/nrf24l01.py

```diff
@@ -144,8 +160,8 @@ def set_channel(self, channel):
     # address should be a bytes object 5 bytes long
     def open_tx_pipe(self, address):
         assert len(address) == 5
-        self.reg_write(RX_ADDR_P0, address)
-        self.reg_write(TX_ADDR, address)
+        self.reg_write(RX_ADDR_P0, buffer=address)
```

Comment:
I think it's important to explicitly say when we are sending an int and when we are sending a buffer.